### ISSUE TITLE
config.py demand update to config3.py using python 3

### ISSUE THREAD

**Issue by amidevous** [emre1393/xtreamui-things#20]
good evening would it be possible to have the config.py script located here

https://github.com/emre1393/xtreamui-things/blob/master/pytools/config.py

in python 3 currently this one is only for python 2 which and obsolete even the dependencies of pip no longer work the pip of the v2 of python and offline

an update to python 3 and therefore urgent

thanks in advance

**Comment by xoceunder** [comment]
The update would be very good since it does not work

**Comment by emre1393** [comment]
yes, update would be very good but i just don't know how to make it work with ubuntu 20 and 22.
anyways, if it is just ubuntu 18 with python 3, i may take a look to install.py

**Comment by emre1393** [comment]
alright, you can use python3 config_p3.py  script if you really need it. ChatGPT adapted functions for python3. 

https://github.com/emre1393/xtreamui-things/blob/master/pytools/config_p3.py

**Comment by xoceunder** [comment]
It would be nice if you update all the python files
@emre1393
